### PR TITLE
Barceloneta LTS

### PR DESCRIPTION
**Pull request by petschki**

### CODE REVIEW

**Comment by petschki** [comment]
@jenkins-plone-org please run jobs

**Review by ale-rt — APPROVED** [review]
"Not-an-Alpine-City-Sprint" Sprint participants agreed to merge the Barceloneta work early to speed up Plone 6 development

**Comment by ale-rt** [comment]
@mauritsvanrees can you force merge this PR? I do not have the power to do that.
Note, test passing in:
- https://jenkins.plone.org/view/PLIPs/job/plip-2967-barceloneta-lts-3.6/
- https://jenkins.plone.org/view/PLIPs/job/plip-2967-barceloneta-lts-3.7/
- https://jenkins.plone.org/view/PLIPs/job/plip-2967-barceloneta-lts-3.8/

**Comment by ale-rt** [comment]
Or @jensens ^

**Comment by ale-rt** [comment]
Note: I would squash the commits

**Comment by ale-rt** [comment]
@jenkins-plone-org please run jobs

**Comment by ale-rt** [comment]
@jenkins-plone-org please run jobs

File: Products/CMFPlone/browser/icons.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+from lxml import etree
```
Comment (tisto) [inline Products/CMFPlone/browser/icons.py:2]: @petschki It seems CMFPlone has lxml as an undeclared dependency. This is one of the imports that currently causes trouble. Though, this undeclared dependency has been added some time ago here:

https://github.com/plone/Products.CMFPlone/commit/946f3b3cee049646d3cf1eb0f176e9cb3c743a54#r60340299

**Pull request merged**